<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game_window</class>
 <widget class="QDialog" name="game_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog {
        background-color: #f0f0f3;
        border-radius: 20px;
    }
    QFrame#frame {
        background-color: #e0e0e0;
        border-radius: 15px;
        border: 1px solid #b0b0b0;
    }
    QLabel#question_label {
        background-color: #f5c211;
        border: none;
        border-radius: 10px;
        font-size: 18px;
        font-weight: bold;
        padding: 10px;
        color: #333333;
    }
    QLabel#question_label_2 {
        background-color: #f5c211;
        border: none;
        border-radius: 10px;
        font-size: 14px;
        padding: 6px;
        font-weight: bold;
        color: #333333;
    }
    QLabel#answer_A_label, QLabel#answer_B_label, QLabel#answer_C_label, QLabel#answer_D_label {
        background-color: #8ff0a4;
        border: none;
        border-radius: 10px;
        padding: 8px;
        font-size: 14px;
        color: #333333;
    }
    QPushButton {
        background-color: #62a0ea;
        color: white;
        border: none;
        border-radius: 10px;
        padding: 8px;
        font-size: 14px;
    }
    QPushButton:hover {
        background-color: #528fcf;
    }
   </string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>520</width>
     <height>400</height>
    </rect>
   </property>
   <widget class="QLabel" name="question_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>480</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>"Question"</string>
    </property>
   </widget>
   <widget class="QWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>480</width>
      <height>150</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="answer_A_label">
       <property name="text">
        <string>"Answer A"</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="answer_B_label">
       <property name="text">
        <string>"Answer B"</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="answer_C_label">
       <property name="text">
        <string>"Answer C"</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="answer_D_label">
       <property name="text">
        <string>"Answer D"</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="question_label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose the right answer:</string>
    </property>
   </widget>
   <widget class="QWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>420</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="A_button">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="B_button">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="C_button">
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="D_button">
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
